### PR TITLE
Remove the extra create ZK node, resulting from merging from release/…

### PR DESCRIPTION
**Pull request by chtyim**
…3.0 branch.

### CODE REVIEW

File: cdap-master/src/main/java/co/cask/cdap/data/runtime/main/MasterServiceMain.java [hunk 1]
```diff
@@ -180,10 +180,6 @@ public void start() {
     metricsCollectionService.startAndWait();
     serviceStore.startAndWait();
     leaderElection.startAndWait();
-
-    // Tries to create the ZK root node (which can be namespaced through the zk connection string)
```
Comment (chtyim) [inline cdap-master/src/main/java/co/cask/cdap/data/runtime/main/MasterServiceMain.java]: It's done in line 176 already.

Comment (anwar6953) [inline cdap-master/src/main/java/co/cask/cdap/data/runtime/main/MasterServiceMain.java]: sorry, I don't understand how the merge to develop didn't remove it from here?

Comment (chtyim) [inline cdap-master/src/main/java/co/cask/cdap/data/runtime/main/MasterServiceMain.java]: Because there was a pick from develop earlier.

**Comment by anwar6953** [comment]
lgtm

**Pull request merged**